libs/s25main/ogl/glSmartBitmap.cpp

```diff
@@ -314,3 +325,18 @@ void glSmartBitmap::drawPercent(DrawPoint drawPt, unsigned percent, unsigned col
     glDrawArrays(GL_QUADS, 0, numQuads);
     glDisableClientState(GL_COLOR_ARRAY);
 }
+
+void glSmartBitmap::drawPercent(DrawPoint drawPt, unsigned percent, unsigned color, unsigned player_color)
```

Comment:
Can you put the `drawPercent` method back above the `draw` method and amend the commit? I know this now matches the order in the header but it makes the diff hard to judge as it looks like you added this method.